app/tranformers/api/v1/appointment_transformer.rb

```diff
@@ -7,7 +7,7 @@ def new_keys_mapping
     end
 
     def to_response(model)
-      h = rename_attributes(model.attributes, inverted_key_mapping)
+      h =  super(model).with_indifferent_access
```

Comment:
```suggestion
      h =  model.with_indifferent_access
```
If I' understanding what `super(model)` returns correctly